flexmeasures/api/v1_1/routes.py

```diff
@@ -62,7 +62,7 @@
 def get_connection():
     """API endpoint to get the user's connections as entity addresses ordered from newest to oldest.
 
-    .. :quickref: User; Retrieve entity addresses of connections
+    .. :quickref: Data; Retrieve entity addresses of connections
```

Comment:
I'd reserve the "Data" quickref for time series data only. "Assets" fits better here.

Reply:
I agree.